src/cassandra.go

```diff
@@ -29,6 +31,7 @@ type argumentList struct {
 	Timeout             int    `default:"2000" help:"Timeout in milliseconds per single JMX query."`
 	ColumnFamiliesLimit int    `default:"20" help:"Limit on number of Cassandra Column Families."`
 	RemoteMonitoring    bool   `default:"false" help:"Identifies the monitored entity as 'remote'. In doubt: set to true."`
+	HideSecrets         bool   `default:"true" help:"Set this to false if you want to see the secrets in the verbose logs."`
```

Comment:
I cannot think a use case in which an end user wants secrets to be logged. Do we really need a flag for it?